.cirrus/tasks_templates.yml

```diff
@@ -81,6 +81,23 @@ chart_testing_on_openshif: &CHART_TESTING_ON_OPENSHIFT_TEMPLATE
     cleanup_script:
       - oc delete project "${VERIFYING_CHART}"
 
+chart_testing_on_kind: &CHART_TESTING_ON_KIND_TEMPLATE
+  <<: *ONLY_ON_NON_RELEASE_DRAFT_TEMPLATE
+  skip: "!changesInclude('charts/**/*', '.cirrus/*')"
```

Comment:
this skip might be useless as it will get override by the child skips, nonetheless it acts as a default that could make sense for other tasks